qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,341 @@
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Trigger Coupling',
+                           unit=None,
+                           initial_value='DC',
+                           val_mapping={'AC': 1,'DC': 2})
+        self.add_parameter(name='external_trigger_range',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Trigger Range',
+                           unit=None,
+                           initial_value='ETR_5V',
+                           val_mapping={'ETR_5V': 0,'ETR_TTL': 2})
+        self.add_parameter(name='trigger_delay',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Trigger Delay',
+                           unit='Sample clock cycles',
+                           initial_value=0,
+                           vals=validators.Multiples(divisor=8, min_value=0))
+        self.add_parameter(name='timeout_ticks',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Timeout Ticks',
+                           unit='10 us',
+                           initial_value=0,
+                           vals=validators.Ints(min_value=0))
+        self.add_parameter(name='aux_io_mode',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='AUX I/O Mode',
+                           unit=None,
+                           initial_value='AUX_IN_AUXILIARY',
+                           val_mapping={'AUX_OUT_TRIGGER': 0,
+                                        'AUX_IN_TRIGGER_ENABLE': 1,
+                                        'AUX_IN_AUXILIARY': 13})
+        self.add_parameter(name='aux_io_param',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='AUX I/O Param',
+                           unit=None,
+                           initial_value='NONE',
+                           val_mapping={'NONE': 0,
+                                        'TRIG_SLOPE_POSITIVE': 1,
+                                        'TRIG_SLOPE_NEGATIVE': 2})
+
+        #The above parameters are important for preparing the card.
+        self.add_parameter(name='mode',
+                           label='Acquisition mode',
+                           unit=None,
+                           initial_value='NPT',
+                           get_cmd=None,
+                           set_cmd=None,
+                           val_mapping={'NPT': 0x200, 'TS': 0x400})
```

Comment:
We have 4 Acquisition modes for the card:
val_mapping={'TM':0x000, 'CM':0x100, 'NPT': 0x200, 'TS': 0x400}

Reply:
We don't expose all modes on the other cards so I think this is fine to leave as is